store/src/java/com/zimbra/cs/zimlet/ZimletUtil.java

```diff
@@ -673,7 +673,20 @@ public static void deployZimletLocally(ZimletFile zf) throws IOException, Zimlet
 
         // install the config
         if (zf.hasZimletConfig()) {
-            installConfig(zf.getZimletConfig());
+            ZimletConfig zc = getZimletConfig(zimletName);
+            if (zc != null) {
+                Version zimletConfigVersion = new Version (zf.getZimletConfig().getVersion().toString());
```

Comment:
should we add try/catch to handle malformed versions?

Reply:
Here we are using version data which is returned by ZimletConfig. Before creating ZimletConfig object all data gets validated. So if any exception comes then we'll get null object.